README.md

```diff
@@ -1956,6 +1956,12 @@ Error while executing script *diskusage.sh
 Error while executing script *cpu.sh
 ```
 
+##### Tailing to console
+
+By default, tailer prints all logs to console (with instance name in front and timestamp converted to the machines time zone). You can simply turn it of:
```

Comment:
I am trying to use rather impersonal forms / sentences without "you" etc/

Reply:
OK